### DISCUSSION TITLE
Change Font Size in the "Use Default Style"

### DISCUSSION THREAD

**Discussion by abounamansour** [neanes/neanes#802] [Q&A] [answered]
I was writing the text in Arabic under the notations, I needed a large font 16 instead of the Default 12.
There is no way to set the size to 16 and write more that one text.
Everytime the Default get re-selected automatically, and the Defaut Font Sizes returns to 12.
I had to make the change on each text under every note, one by one.
So: Either prevent the Use Default Style to get selected everytime
Or allow us to set the needed Font Size in the Default.

Excellent Job! God Bless!
Fr. Mansour Azar

**Comment by danielgarthur** [comment, marked as answer]
In the `Page Setup` dialog, the default lyric size can be changed. Is this what you are looking for?
<img width="241" alt="image" src="https://github.com/neanes/neanes/assets/4132779/362664d6-6607-473e-bf21-97044ef66d3b">

From the main menu, `File -> Page Setup`.

**Comment by abounamansour** [comment]
Oh yes!

I was changing it from cleaning the selection of Use Default Style.
I did never see the Page setup.
This solves the problem for me.
Thank you for your help!
God Bless!
Fr. Mansour
